From cba3f090b19cd02ad4f3140708d6f5444bf6a7d9 Mon Sep 17 00:00:00 2001
From: Martin Weinelt <martin@darmstadt.freifunk.net>
Date: Fri, 5 Apr 2019 01:39:37 +0200
Subject: [PATCH] modules: update routing

e5fe4b6 Merge pull request #462 from ecsv/batadv-18.06
ee2d981 batman-adv: Merge bugfixes from 2019.1
4d7a182 nodogsplash: fix invalid pointer bug when clock is turned back (#456)
2ad165c Merge pull request #452 from dangowrt/openwrt-18.06
71f9aae luci-app-bmx7: update to v0.1-alpha
0e3d701 bmx7: update to git snapshot as of 2018-12-29
42af835 batman-adv: Refresh patches

(cherry picked from commit b6c4e06b4032b5df27d0eeede9ad0d15db1d6863)
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 59fc8be..0549f9a 100644
--- a/modules
+++ b/modules
@@ -10,7 +10,7 @@ PACKAGES_PACKAGES_COMMIT=b7bcf24a00153f0c99ff2e01fa33876680c6128d
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt-routing/packages.git
 PACKAGES_ROUTING_BRANCH=openwrt-18.06
-PACKAGES_ROUTING_COMMIT=ea345d16a6e27c2a8fdf67bf543cc36a5f189131
+PACKAGES_ROUTING_COMMIT=e5fe4b63e8063d87ccbdc9044f77e5f57e9f5eed
 
 PACKAGES_LUCI_REPO=https://github.com/openwrt/luci.git
 PACKAGES_LUCI_BRANCH=openwrt-18.06
-- 
2.7.4

